contracts/light-clients/SP1ICS07Tendermint.sol

```diff
@@ -428,16 +428,19 @@ contract SP1ICS07Tendermint is ISP1ICS07TendermintErrors, ISP1ICS07Tendermint, I
     /// @notice Validates the client state and time.
     /// @dev This function does not check the equality of the latest height and isFrozen.
     /// @param publicClientState The public client state.
-    /// @param time The time.
+    /// @param time The time in unix nanoseconds.
     function _validateClientStateAndTime(
         IICS07TendermintMsgs.ClientState memory publicClientState,
-        uint64 time
+        uint128 time
```

Comment:
why not uint256? Seems like we are using that elsewhere here

Reply:
rust returns nanos as `u128`. I think this is a nice reminder to us that this field is a nanos. Making us less likely to confuse.